main.py

```diff
@@ -116,22 +150,22 @@ def main(mode, config):
                 total_loss += loss.item()
                 total_metric += metric(output, target)
 
-            training_log["loss"].append(total_loss / len(training_data_loader))
-            training_log["metric"].append(total_metric / len(training_data_loader))
+            train_log["loss"].append(total_loss / len(train_data_loader))
+            train_log["metric"].append(total_metric / len(train_data_loader))
 
             if epoch % config["checkpoint_frequency"] == 0:
                 torch.save(model.state_dict(), checkpoint_path_template.format(epoch))
 
                 print(f"epochs {epoch}")
-                print(f"\tloss: {training_log['loss'][-1]}")
-                print(f"\tmetric: {training_log['metric'][-1]}")
+                print(f"\tloss: {train_log['loss'][-1]}")
```

Comment:
String single vs double quotation convention should be consistent within a project.

Reply:
ya, right, I should keep it consistent